cms/templates/widgets/problem-edit.html

```diff
@@ -159,4 +163,15 @@ <h6>${_("Explanation")}</h6>
     </article>
 </script>
 </div>
+<div id="visual-editor" class="visual-editor is-inactive"></div>
+<script>
+    var metaData = ${editable_metadata_fields| n, dump_js_escaped_json};
```

Comment:
Is this being used? We shouldn't be adding a variable with a name as vague as `metaData` to the global scope.

Can you instead just write

```
LXCData.max_attempts = ${editable_metadata_fields.max_attempts | n, dump_js_escaped_json};
```
or something like that? There must be some way to do that and get just the variable you need.

Reply:
done